src/cirrus/cirrus_setup.py

```diff
@@ -10,11 +10,11 @@
 
 from argparse import ArgumentParser
 
-from cirrus.configuration import load_setup_configuration, get_creds_plugin
+from cirrus.configuration import load_setup_configuration, get_github_api_base
 from cirrus.logger import get_logger
 
 LOGGER = get_logger()
-GITHUB_AUTH_URL = "https://api.github.com/authorizations"
+GITHUB_AUTH_URL = get_github_api_base()
```

Comment:
Looks like this is only used in `create_github_token`. Can we remove the constant and just call `get_github_api_base` there?